source/Octopus.Tentacle/Commands/ClientModule.cs

```diff
@@ -11,7 +11,11 @@ protected override void Load(ContainerBuilder builder)
         {
             base.Load(builder);
             builder.RegisterType<OctopusClientFactory>().AsSelf().AsImplementedInterfaces();
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
It might be worth noting why this is obsolete, and if/when it could be removed or if instead it must remain forever for some reason.